packages/react/src/components/Radio/Radio.module.css

```diff
@@ -0,0 +1,117 @@
+.center {
+  display: inline-flex;
+  align-items: center;
+}
+
+.container {
+  position: relative;
+  padding-left: calc(var(--fds-spacing-6) + 20px);
+}
+
+.icon {
+  grid-area: input;
+  pointer-events: none;
+  height: 1.75em;
+  width: 1.75em;
+}
+
+.label {
+  min-height: 44px;
+  min-width: min-content;
+  display: inline-flex;
+  flex-direction: row;
+  gap: var(--fds-spacing-1);
+  align-items: center;
+  cursor: pointer;
+}
+
+.description {
+  color: var(--fds-semantic-text-neutral-subtle);
+}
+
+.radio {
+  position: absolute;
+  left: 0;
+  top: 0;
+  min-width: 44px;
+  min-height: 44px;
+  display: inline-grid;
+  grid: [input] 1fr / [input] 1fr;
+  gap: var(--fds-spacing-2);
+  grid-auto-flow: column;
+  border-radius: var(--fds-border_radius-full);
+}
+
+.radio input {
+  height: 100%;
+  width: 100%;
+  opacity: 0;
+  margin: 0;
+  cursor: pointer;
+  grid-area: input;
+}
+
+.radio svg {
+  border-radius: var(--fds-border_radius-full);
+  z-index: 1;
+  margin: auto;
+}
+
+.container:has(input:disabled) .label,
```

Comment:
Can you check if this works within Firefox? Im not sure if `:has` is supported by default with Firefox.

Reply:
I've understood its partially supported. I tested now and this selector didn't work in firefox :(
Will explore other options or fallbacks 🤔